### PR TITLE
feat: delete remote link by id

### PR DESCRIPTION
**Pull request by DevXiaolan**

### CODE REVIEW

**Review by pioug — CHANGES_REQUESTED** [review]
There are many unnecessary changes. We can't merge it as it is. Please review your PR and focus solely on relevant changes.

**Comment by DevXiaolan** [comment]
Sorry about my mistakes

File: package-lock.json [hunk 1]
```diff
@@ -6441,4 +6441,4 @@
       }
     }
   }
-}
+}
```
Comment (pioug) [inline package-lock.json:6444]: There is still this tiny tiny change, can you undo it @DevXiaolan? 🙏

**Review by pioug — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (DevXiaolan) [reply package-lock.json]: Fix it

**Comment by DevXiaolan** [comment]
well，
What are the conditions for entering the list of contributors? :)

**Comment by pioug** [comment]
You featured as a contributor in the release notes https://github.com/jira-node/node-jira-client/releases/tag/v7.2.0.

If you mean being members of the repo, I think I didn't give out an invitation because your PR wasn't good initially and it took 9 commits to get it right. Sorry for judging hard, but I am cautious because it's not really my project. You can keep making PRs, I'll do my best to review and merge them.

**Pull request merged**